src/main/java/teammates/ui/template/FeedbackResultsTable.java

```diff
@@ -24,7 +26,45 @@ public FeedbackResultsTable(int fbIndex, String studentName, FeedbackSessionResu
             for (Map.Entry<String, List<FeedbackResponseAttributes>> entry : received.entrySet()) {
                 giverIndex++;
                 this.receivedResponses.add(new FeedbackResponsePersonRow(fbIndex, giverIndex, entry.getKey(), "giver",
-                                                                         entry.getValue(), result));
+                        entry.getValue(), result));
+            }
+
+            boolean hasAnsweredContrib = false;
+            boolean contribExistsFromOtherStudent = false;
+            FeedbackResponseAttributes response = new FeedbackResponseAttributes();
+
+            for (Map.Entry<String, List<FeedbackResponseAttributes>> entry : received.entrySet()) {
+                for (int j = 0; j < entry.getValue().size(); j++) {
```

Comment:
Use `map.forEach((key, value) -> { ... })` and `for (String s : list)` to make the code more readable.

Reply:
`map.forEach((key, value) -> { ... })` is throwing error `variable used in lambda expression should be final or effectively final` but I need to update the local variables inside, so I am keeping it as it is. Finding a work around this could make the logic more confusing.
I will keep this in mind in future contributions.

Reply:
Making `hasAnsweredContrib` and `contribExistsFromOtherStudent` an array of size one will bypass that limitation without too much effect on readability.